SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLatencyTracer.h

```diff
@@ -69,6 +69,10 @@ class SPATIALGDK_API USpatialLatencyTracer : public UObject
 	UFUNCTION(BlueprintCallable, Category = "SpatialOS", meta = (WorldContext = "WorldContextObject"))
 	static void RegisterProject(UObject* WorldContextObject, const FString& ProjectId);
 
+	// Set a prefix to be used for all span names. Resulting uploaded span names are of the format "PREFIX(WORKER_ID) : USER_SPECIFIED_NAME".
+	UFUNCTION(BlueprintCallable, Category = "SpatialOS", meta = (WorldContext = "WorldContextObject"))
```

Comment:
Can we make this an optional argument for RegisterProject? Regular users won't really need to know about this

Reply:
Setting a prefix isn't really related to registering a project, would it be appropriate to add it there?

Reply:
You're passing a world context object here, but aren't using it in the function. I think you should to get a handle on this world's latency tracer, then set the message on the tracer itself.

Reply:
As brought up earlier by Danny and suggested by you, I've made the prefix a member. Please TAL